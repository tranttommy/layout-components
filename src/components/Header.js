import React from 'react';
import logo from '../../assets/logo-black.png';

export default function Header() {
  return (
    <header>
      <img src={ logo } />
      <h1>Simple Dog</h1>
    </header>
  );
}
