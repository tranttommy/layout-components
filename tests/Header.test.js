import React from 'react';
import { shallow } from 'enzyme';
import Header from '../src/components/Header.js';

describe('Header component', () => {
  it('renders a Header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
