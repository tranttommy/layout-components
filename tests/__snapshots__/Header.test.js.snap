// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders a Header 1`] = `
<header>
  <img
    src="/filepath"
  />
  <h1>
    Simple Dog
  </h1>
</header>
`;
